src/main/java/controller/FileController.java

```diff
@@ -0,0 +1,59 @@
+package controller;
+
+import domain.AttendanceManager;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+
+public class FileController {
+
+    private final String FILE_ERROR_MESSAGE = "파일을 불러오는 데 문제가 발생했습니다. 잠시 후 다시 시도해 주세요.";
+    private AttendanceManager attendanceManager;
```

Comment:
```suggestion
    private static final String FILE_ERROR_MESSAGE = "파일을 불러오는 데 문제가 발생했습니다. 잠시 후 다시 시도해 주세요.";
    
    private final AttendanceManager attendanceManager;
```

상수는 static으로, 인스턴스 변수는 final로 수정할까요? 두 키워드를 왜 붙여야 할까요? 전체적으로 누락된 곳이 많아 점검 부탁드립니다.
또 상수와 변수 사이는 한 줄 띄워주면 더 구분하기 좋습니다.

Reply:
상수를 일관되게 사용하기 위해 static 키워드를 사용하고 상수의 목적을 구분하기 위해 클래스로 분리하도록 수정했습니다.